comps/dataprep/utils.py

```diff
@@ -163,6 +214,7 @@ def load_svg(svg_path):
     return text
 
 
+@timeout(600)
```

Comment:
@lvliang-intel  pls help to check if this part is ok for you?

@xuechendi my major concern here is even timeout reaches, there is no benefit for user as the current operation will only dump message in server side, user aren't aware of this.

Reply:
@ftian1, 
1. for file size less than 10, it will go python mode, and any failure including timeout will be return in error 400 description. 
2. For any file size more than 10, it will go ray mode, yes, only server side captures the info of which files are failed. Since if we are handling 1000 or more files, returning such info to user may not be necessary.
(But if you would like to add that, I can make it return 200 but provide filename which is not successfully handled in description.If you think this works for you, I'll add this feature.)

Reply:
I just think the purpose of introducing timeout is used to capture those hang operation, right? shall it be deal with by k8s or by self? I am worried that there always has a case in which the file size is huge and request longer time than your hardcoded timeout value in code.

Reply:
@ftian1, yes, most of distributed framework(ray, spark, etc) will provide a timeout mechanism for hanging tasks or tasks longer than expectation. 10min is a reasonable setting, if a single file/url processing takes more than 10 min, I think we should ask user to split or we should provide special treatment for it?
And for ray, if the timeout is caused by bad resource allocation, we can rerun the task based on the log.

I would suggested if we can test with actual enterprise usecase, we can make timeout configurable and also provide failover resume based on the user scenario.

Reply:
@ftian1 , I added status return and provide timeout as a configurable environment.
* status will return to client
![image](https://github.com/opea-project/GenAIComps/assets/4355494/5dd638d7-e4ee-4131-afd6-1699d986797c)

* Timeout setting
![image](https://github.com/opea-project/GenAIComps/assets/4355494/b524fe9b-1d15-4034-9e3a-d7869a152018)